controllers/offer.js

```diff
@@ -15,21 +15,14 @@ const getOfferById = async (req, res) => {
 }
 
 const createOffer = async (req, res) => {
-  const userId = req.params.id
-  const userRole = req.user.role
-  const { price, proficiencyLevel, description, languages, subjectId, categoryId, isActive } = req.body
-
-  const newOffer = await offerService.createOffer(
-    price,
-    proficiencyLevel,
-    description,
-    languages,
-    userRole,
+  const { id: userId } = req.params
+  const authorRole = req.user.role
```

Comment:
forgot to destructuring this too:
const { authorRole } = req.user.role

Reply:
> forgot to destructuring this too: const { authorRole } = req.user.role

const { role: authorRole } = req.user

Reply:
you're right, my bad

Reply:
fixed